.github/workflows/integration-test.yml

```
@@ -18,11 +21,16 @@ jobs:
         with:
           ruby-version: '3.1.2'
 
-      - name: Install sentry-cli
-        run: curl -sL https://sentry.io/get-cli/ | bash
-
-      - name: Install Fastlane
+      - if: matrix.os == 'macos-latest'
+        name: Install Fastlane (macOS)
```

`l`: You can also use the matrix in the name, so you don't have to duplicate this.
```suggestion
      - if: matrix.os == 'macos-latest'
        name: Install Fastlane {{ matrix.os }}
```